src/evm/decoder.rs

```diff
@@ -50,6 +50,8 @@ pub(super) struct TychoStreamDecoder {
     inclusion_filters: HashMap<String, FilterFn>,
 }
 
+/// Decodes a `FeedMessage` into a `BlockUpdate` containing the updated states of protocol
+/// components for a specific block.
```

Comment:
Could you please convert this to have short heading and then more descriptive paragraph of text under it.

Reply:
```suggestion
// Decodes incoming raw messages
//
// This struct decodes incoming messages of type `FeedMessage` and converts it into the `BlockUpdate`struct...
```